### DISCUSSION TITLE
Pointers on understanding log content

### DISCUSSION THREAD

**Discussion by Hugoagogo** [gopro/labs#550] [Q&A]
Is there any documentation of what the different logs that end up in the log file mean.

I have the labs firmware installed (1.72.70 I believe) but have been trying to run some general timelapses of some renovation work.

The gopro has been connected to power, but has been stopping recording after an hour or so, it worked fine for the first few days of recording so I am not sure whats up.

I was poking around on the SD card and found the below logs, but cant find anything on understanding what these messages might indicate if anything.

[qrlog.txt](https://github.com/gopro/labs/files/12659311/qrlog.txt)

**Comment by dnewman-gpsw** [comment]
Is this a HERO9?

Are you running a Labs script that is failing?  The Labs qrlog.txt will be for only Labs commands, so it will not report anything about the general functionality of the camera.  It is only used to debug Labs scripting.

**Reply by Hugoagogo** [reply]
Yes this is a 9, just running the stock time-lapse.

I understand the log will be a bit sparse because I am not running a script but I had hoped somebody might have some insight on what the shutdown:in progress might be caused by and what the ext_type lines mean.

**Reply by dnewman-gpsw** [reply]
those ext_type log extries are a logging bug, they can be ignored.

**Reply by Hugoagogo** [reply]
Are there any ways to put the GoPro into a mode where it will log more heavily to help identify the cause of stopped recordings.

**Reply by dnewman-gpsw** [reply]
Only is you are using Labs scripts, and you not from what I can see -- Stock timelapse will not report anything to Labs.
